generators/constants.js

```diff
@@ -1,37 +1,49 @@
+const LOCALSTACK_IMAGE_VERSION = '2.2';
+const POSTGRESQL_IMAGE_VERSION = '15.4-alpine';
+const MARIADB_IMAGE_VERSION = '11.0.3';
+const MYSQL_IMAGE_VERSION = '8.1.0';
+
 module.exports = {
-    JAVA_VERSION : '17',
-    SPRING_BOOT_VERSION : '3.1.1',
-    SPRING_CLOUD_VERSION : '2022.0.3',
-    SPRING_CLOUD_AWS_VERSION : '3.0.1',
-    SPRING_DEP_MNGMNT_VERSION : '1.1.0',
-    DEFAULT_APP_VERSION : '0.0.1-SNAPSHOT',
-    TEST_CONTAINERS_VERSION : '1.18.3',
-    SPRINGDOC_OPENAPI_VERSION : '2.1.0',
-    COMMONS_IO_VERSION : '2.13.0',
-    LOGSTASH_LOGBACK_ENCODER : 7.3,
-
-    JACOCO_MIN_COVERAGE_REQUIRED:'0.80',
-
-    GRADLE_GIT_PROPERTIES_PLUGIN_VERSION:'2.4.1',
-    GRADLE_JACOCO_PLUGIN_VERSION:'0.8.10',
-    GRADLE_SONAR_PLUGIN_VERSION:'4.2.1.3168',
-    GRADLE_OWASP_PLUGIN_VERSION:'8.2.1',
-    GRADLE_BENMANES_VERSIONS_PLUGIN_VERSION:'0.47.0',
-    SPOTLESS_PLUGIN_VERSION:'6.19.0',
-
-    MAVEN_DEPENDENCY_CHECK_PLUGIN_VERSION:'8.2.1',
-    MAVEN_PROPERTIES_PLUGIN_VERSION:'1.1.0',
-    MAVEN_SUREFIRE_PLUGIN_VERSION:'3.1.2',
-    MAVEN_FAILSAFE_PLUGIN_VERSION:'3.1.2',
-    MAVEN_SONAR_PLUGIN_VERSION:'3.9.1.2184',
-    MAVEN_JACOCO_PLUGIN_VERSION:'0.8.10',
-    MAVEN_SPOTLESS_PLUGIN_VERSION:'2.37.0',
-
-    KEY_FLYWAY_MIGRATION_COUNTER : 'flywayMigrationCounter',
-    KEY_LIQUIBASE_MIGRATION_COUNTER : 'liquibaseMigrationCounter',
-
-    LOCALSTACK_IMAGE: 'localstack/localstack:2.1.0',
-    POSTGRESQL_IMAGE: 'postgres:15.3-alpine',
-    MARIADB_IMAGE: 'mariadb:11.0',
-    MYSQL_IMAGE: 'mysql:8.0.3',
+    JAVA_VERSION: '17',
+    SPRING_BOOT_VERSION: '3.1.3',
+    SPRING_CLOUD_VERSION: '2022.0.4',
+    SPRING_CLOUD_AWS_VERSION: '3.0.2',
+    SPRING_DEP_MNGMNT_VERSION: '1.1.3',
+    DEFAULT_APP_VERSION: '0.0.1-SNAPSHOT',
+    TEST_CONTAINERS_VERSION: '1.19.0',
```

Comment:
Should we manage test containers version manually? I prefer using framework supported version